references/proxy-template/generate-proxy.sh

```diff
@@ -13,16 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 set -e
 
-
 # Ask for input parameters if they are not set
 
 [ -z "$PROXY" ]     && printf "Proxy Name: "    && read -r PROXY
 [ -z "$VERSION" ]   && printf "Proxy Version: " && read -r VERSION
 [ -z "$VHOST" ]     && printf "Virtual Host: "  && read -r VHOST
-[ -z "$TARGETURL" ] && printf "Target URL: "    && read -r TARGETURL
+[ -z "$TARGET_SERVER_NAME" ] && printf "Target Server Name: "    && read -r TARGET_SERVER_NAME
+[ -z "$TARGET_PATH" ] && printf "Target Path: "    && read -r TARGET_PATH
```

Comment:
Not sure I fully understand the mechanics here. There is a TARGET_PATH variable set by the user explicitly here and one by splitting the TARGET_URL in the `set-targetserver-envs.sh` isn't one the proxy path suffix?

Reply:
Yes I agree Daniel, this is not clear